typing/ctype.ml

```diff
@@ -2664,6 +2664,7 @@ let complete_type_list ?(allow_absent=false) env fl1 lv2 pack2 =
 
      It'd be nice if we avoided creating such temporary dummy modules and broken
      environments though. *)
+  let exception Missing of string list in
```

Comment:
Ah, I wish one could just write `let exception missing n = Error n in ...`. ( #11740 )